src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  let discount = 0;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    discount = SHORT_TERM_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    discount = LONG_TERM_DISCOUNT;
+  } else {
+    discount = 0;
+  }
```

Comment:
The 'else' clause on line 17 is unnecessary because both the 'if' and 'else if' clauses on lines 13 and 15 have return statements. The code can be simplified by removing the 'else' clause and setting the default discount to 0 before the conditional statements.